### ISSUE TITLE
[Bug] Flag `-t` is shown as possible alias for `--log-timestamp` flag in `compile` CLI commmand

### ISSUE THREAD

**Issue by Luna-Klatzer** [Kipper-Lang/Kipper#451]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### I am following the documentation's guide

- [X] I have read the documentation

### This issue exists in the latest version

- [X] I am using the latest version

### Current Behavior

When typing `kipper compile --help` the following list of flags is shown:

```
  -b, --[no-]optimise-builtins   Optimise the generated built-in functions using tree-shaking to reduce the size of the output.
  -e, --encoding=encoding        [default: utf8] The encoding that should be used to read the file (ascii,utf8,utf16le).
  -i, --[no-]optimise-internals  Optimise the generated internal functions using tree-shaking to reduce the size of the output.
  -o, --output-dir=output-dir    [default: build] The build directory where the compiled files should be placed. If the path does not exist, it will be created.
  -s, --string-code=string-code  The content of a Kipper file that can be passed as a replacement for the 'file' parameter.
  -t, --[no-]log-timestamp       Show the timestamp of each log message.
  -t, --target=js|ts             [default: js] The target language where the compiled program should be emitted to.
  -w, --[no-]warnings            Show warnings that were emitted during the compilation.
  --[no-]abort-on-first-error    Abort on the first error the compiler encounters.
  --[no-]recover                 Recover from compiler errors and log all detected semantic issues.
```

In that list `-t` is shown as a valid alias for the flag `--[no-]log-timestamp` which should not be the case.

### Expected Behavior

The help guide should display the help guide that is the same as `run`, as the flags are as of `v0.10.0` the same:

```
  -b, --[no-]optimise-builtins   Optimise the generated built-in functions using tree-shaking to reduce the size of the output.
  -e, --encoding=encoding        [default: utf8] The encoding that should be used to read the file (ascii,utf8,utf16le).
  -i, --[no-]optimise-internals  Optimise the generated internal functions using tree-shaking to reduce the size of the output.
  -o, --output-dir=output-dir    [default: build] The build directory where the compiled files should be placed. If the path does not exist, it will be created.
  -s, --string-code=string-code  The content of a Kipper file that can be passed as a replacement for the 'file' parameter.
  -t, --target=js|ts             [default: js] The target language where the compiled program should be emitted to.
  -w, --[no-]warnings            Show warnings that were emitted during the compilation.
  --[no-]abort-on-first-error    Abort on the first error the compiler encounters. Same behaviour as '--no-recover'.
  --[no-]log-timestamp           Show the timestamp of each log message.
  --[no-]recover                 Recover from compiler errors and display all detected compiler errors.
```

### Steps To Reproduce

1. Call `kipper compile --help` in a terminal.

### Environment

- Kipper: 0.10.1
- Environment: Terminal
- Operating System: Arch Linux with Linux 6.1.23-1-lts

**Comment by Luna-Klatzer** [comment]
Fixed by #479 and will be released in the next patch version of the latest stable release (`v0.10.3`).

**Comment by Luna-Klatzer** [comment]
To be implemented as well for `0.11.0-alpha.1` ([`#510`](https://github.com/Kipper-Lang/Kipper/pull/510)).

**Comment by Luna-Klatzer** [comment]
Finished issue with merge of #510.
